packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -208,31 +207,41 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
     const onRenderPlaceholder = props.onRenderPlaceholder || props.onRenderPlaceHolder || this._onRenderPlaceholder;
 
     const selectedOptions = getAllSelectedOptions(options, selectedIndices);
+    const selectedOptionsAriaLabel = this._joinSelectedOptions(selectedOptions);
     const divProps = getNativeProps(props, divProperties);
 
     const disabled = this._isDisabled();
 
     const optionId = id + '-option';
-    const ariaAttrs =
-      multiSelect || disabled
-        ? {
-            role: undefined,
-            ariaActiveDescendant: undefined,
-            childRole: undefined,
-            ariaSetSize: undefined,
-            ariaPosInSet: undefined,
-            ariaSelected: undefined
-          }
-        : // single select
-          {
-            role: 'listbox',
-            ariaActiveDescendant:
-              isOpen && selectedIndices.length === 1 && selectedIndices[0] >= 0 ? this._id + '-list' + selectedIndices[0] : optionId,
-            childRole: 'option',
-            ariaSetSize: this._sizePosCache.optionSetSize,
-            ariaPosInSet: this._sizePosCache.positionInSet(selectedIndices[0]),
-            ariaSelected: selectedIndices[0] === undefined ? undefined : true
-          };
+    const ariaAttrs = disabled
```

Comment:
Nit: at this point I feel like this logic should be put into a function somewhere. It's a bit big to have in the render. Not a blocker though.

Reply:
Good call!